### PR TITLE
Feature/edit text single planet

### PR DESCRIPTION
**Pull request by alexthompson207**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [X] Feature
- [X] Refactor
- [X] Style
- [ ] Testing

### Are there any known bugs?

- [X] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Add state to `PlanetInfo` component, and add an input field to the gravity fact so that a user can input a weight on earth and see how much that would weigh on the respective planet they are viewing.

### What ticket(s) do the changes resolve?
#39 

Please Link Issues to this PR as necessary.

### Notes for the Reviewer

This is just basic styling, definitely okay to change as needed. 

### How to Test Changes?

Pull down this branch, should be no errors in the console when viewing the website locally. When clicking on a planet should see a single planet view with a default of 100 pounds on earth is  XX pounds on the respective planet. Users should be able to input whatever positive integer they wish in this field and see changes render instantly.

### Next Iterations
1. Make this view and input section responsive.
2. Test this input functionality.
3. Style this to match Katie's branch.

### CODE REVIEW

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
```
Comment (knees4bees) [inline src/components/PlanetInfo/PlanetInfo.tsx:1]: Yeah Hooks!!!

File: src/components/PlanetInfo/PlanetInfo.tsx [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
-import { PlanetBio, InfoProps } from '../../interface';
+import { InfoProps } from '../../interface';
 
 const PlanetInfo: React.FC<InfoProps> = ({ currentPlanet }): JSX.Element => {
+  const [input, setInput] = useState('100');
 
   const { name, mass, diameter, gravity, length_of_day, distance_from_sun, length_of_year, number_of_moons } = currentPlanet;
 
+  const gravityConversion = (planetGravity: number | string): number => {
+    const weightOnPlanet = Math.round((Number(planetGravity) / 9.807) * Number(input));
```
Comment (knees4bees) [inline src/components/PlanetInfo/PlanetInfo.tsx:12]: Such a nice, simple way to calculate this. Good job.

**Review by knees4bees — APPROVED** [review]
Heck yeah, this is great work.

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks good! Nice work getting that function off the ground.

**Pull request merged**